packages/nvidia_nat_opentelemetry/src/nat/plugins/opentelemetry/register.py

```
@@ -193,3 +193,45 @@ async def galileo_telemetry_exporter(config: GalileoTelemetryExporter, builder:
+
+
+class DBNLTelemetryExporter(BatchConfigMixin, TelemetryExporterBaseConfig, name="dbnl"):
+    """A telemetry exporter to transmit traces to DBNL."""
+
+    api_url: str | None = Field(description="The DBNL API url.", default=None)
+    api_token: str | None = Field(description="The DBNL API token.", default=None)
+    project_id: str | None = Field(description="The DBNL project id.", default=None)
+
+
+@register_telemetry_exporter(config_type=DBNLTelemetryExporter)
+async def dbnl_telemetry_exporter(config: DBNLTelemetryExporter, builder: Builder):
```

Ruff (0.14.1)

Unused function argument: `builder`

(ARG001)